scripts/proto_splitter/proto2mesh.py

```diff
@@ -449,18 +551,80 @@ def write_obj(self, meshes):
         default=None,
         help='Specifies the proto file, or a directory. Converts all .proto files, if it is a directory.',
     )
+    optParser.add_option(
+        '--output',
+        dest='outPath',
+        default=None,
+        help='Specifies the output path. Only for directory conversions.',
+    )
+    optParser.add_option(
+        '--mt', '--multithreaded',
+        dest='mt',
+        default=False,
+        action='store_true',
+        help='If set, enables multicore processing for directories with several PROTO files. May not work on Windows.',
+    )
+    optParser.add_option(
+        '--verbose', '-v',
+        dest='verbose',
+        default=False,
+        action='store_true',
+        help='If set, detailed output of mesh conversion is shown in console',
+    )
+    optParser.add_option(
+        '--check-protos-validity', '--cpv',
+        dest='cpv',
+        default=False,
+        action='store_true',
+        help='If set, will quickly go through all protos files and output any errors. No mesh calculations or files created.',
+    )
     options, args = optParser.parse_args()
     inPath = options.inPath
-    if inPath is not None:
-        p2m = proto2mesh()
-        if os.path.splitext(inPath)[1] == '.proto':
-            p2m.convert(inPath)
-            print('Done')
-        elif os.path.isdir(inPath):
-            inPath = os.path.abspath(inPath)
-            p2m.convert_all(inPath)
-            print('Done')
-        else:
-            sys.exit('Error: --input has to be a .proto file or directory!')
+    outPath = options.outPath
+    verbose = options.verbose
+    cpv = options.cpv
```

Comment:
We should try to avoid this kind of abbreviation in variable and function names, prefer `checkProtoValidity` instead.